docs/scos/dev/glue-api-guides/202307.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.md

```diff
@@ -26,7 +26,7 @@ The `spy_dynamic_entity_configuration` table represents the configuration for dy
 
 {% info_block infoBox %}
 
-Currently, the process entails manually executing SQL queries as there is no existing user interface (UI) feature in Spryker for it. 
+Currently, the process entails manually executing SQL queries or importing via config file as there is no existing user interface (UI) feature in Spryker for it. 
```

Comment:
don't use "via"
Use "because" instead of "as"
```suggestion
The process entails manually executing SQL queries or importing using a config file because there is no existing user interface (UI) feature in Spryker for it. 
```